src/main/java/com/example/bookstore/config/AppConfig.java

```diff
@@ -0,0 +1,9 @@
+package com.example.bookstore.config;
+
+import org.springframework.context.annotation.ComponentScan;
+import org.springframework.context.annotation.Configuration;
+
+@Configuration
+@ComponentScan(basePackages = "com.example.bookstore")
```

Comment:
Component scan is embedded into `SpringBootApplication` annotation, so this config is redundant.